packages/libraries/cli/src/commands/artifact/fetch.ts

```diff
@@ -48,6 +48,18 @@ export default class ArtifactsFetch extends Command<typeof ArtifactsFetch> {
       retry: {
         retries: 3,
       },
+      logger: {
+        info: (...args) => {
+          // Note:
+          // We log to std:error here to avoid piped stdout from being polluted.
+          if (this.flags.debug) {
```

Comment:
Why is info an error?

Reply:
For this command specifically, because some people run

```
hive artifact:fetch > foo.graphql
```

instead of

```
hive artifact:fetch --outputFile foo.graphql
```

If we write to std:error - we still support `--debug` for these people, as std:error would not be piped to the file. But I am also fine with writing to std:out. The bug earlier was due to us accidentally logging even if the `--debug` flag is not provided.

Reply:
But that’s my problem with it. Debug is special and whoever runs —debug should know what’s the deal with it.
If you curl to a file, you won’t do a verbose mode or something

Reply:
ok will change to info then

Reply:
done!